src/lib.rs

```diff
@@ -423,6 +426,48 @@ impl Account {
         })
     }
 
+    /// Load a new account by private key, with a default or custom HTTP client
```

Comment:
I don't think this headline is accurate for this fn: it always uses the default HTTP client.